### DISCUSSION TITLE
Form - (keyboardShortcuts: disable) - Textarea - Prevent submit form on Return key

### DISCUSSION THREAD

**Discussion by brunotourinho** [fomantic/Fomantic-UI#857] [Behaviors (Api, Form Validation, Visibility)] [answered]
Hey guys, I've read something about disabling form submit hitting Enter/Return key.
My problem is, in my form `<form>` I have a couple of textareas, when the user is typing some long text and hit enter, the form quietly (without validation, and no errors) submits once, then when click on submit button, it submits again. 

This "bug" is causing duplicated entries in my backend.

I've tried setting: keyboardShortcuts: disable on the form, but it doesn't seem to work.

Can you help me?

**Comment by prudho** [comment, marked as answer]
Please try `keyboardShortcuts: false` into your form settings.

**Comment by brunotourinho** [comment]
I'll take a shot!
Can I make it global? with `$.fn.form.settings` or something?

**Comment by lubber-de** [comment]
@brunotourinho Yes: use `$.fn.form.settings.keyboardShortcuts = false;` after import of `semantic.js` or `form.js` but before any instantiation of any form component.

**Comment by lubber-de** [comment]
@brunotourinho  Please tell us, if this is solving your issue. If not, please provide a jsfiddle for more investigation. Thank you 😉

**Comment by brunotourinho** [comment]
It worked like a charm!
Thank you guys! 😁
